src/App.js

```diff
@@ -7,6 +7,7 @@ import List from './pages/List';
 import Settings from './pages/Settings';
 import Nav from './components/Nav';
 import './App.css';
+import LoadingMessage from './components/LoadingMessage';
```

Comment:
Loading/error indicators weren't specified in the ticket, but since it's done we can keep it. In the future, this is probably something to raise in the team channel to ask for guidance on what to do. E.g., you could ask whether it's OK to add this scope to the current ticket or whether your product owner would want a separate ticket to track this work. On a project with a tight deadline, this might be UI polish that your product person decides is acceptable to ship without, at least at first.